### ISSUE TITLE
Performance way too slow

### ISSUE THREAD

**Issue by danilw** [keith2018/spvm#28]
I think this project spvm does not use "SIMD vector optimizations" this why performance is slow.

**Maybe adding SIMD optimizations can be added to TODO of this project.**

**Result of this project spvm** `spvm_shadertoy` XdlSDs.frag - **5 fps**
![2](https://user-images.githubusercontent.com/24825887/190913107-7855a25f-cbfc-485c-89ea-13d3112155ce.png)

Result of **llvmpipe** (CPU) - on 1280x720 resolution **160-190 fps**
![1](https://user-images.githubusercontent.com/24825887/190913141-572e85ec-c872-46de-991c-cba894084170.png)

**Comment by keith2018** [comment]
Yes, optimizations such as SIMD/JIT/AOT should be implemented to make it more faster, it is in the TODO list now
